src/ID3Helpers.js

```diff
@@ -96,55 +96,61 @@ module.exports.getTagsFromBuffer = function(filebuffer, options) {
     return getTagsFromFrames(frames, ID3Version, options)
 }
 
-function getFramesFromID3Body(ID3FrameBody, ID3Version, options = {}) {
+function isFrameDiscardedByOptions(frameIdentifier, options) {
+    if(options.exclude instanceof Array && options.exclude.includes(frameIdentifier)) {
+        return true
+    }
+
+    return options.include instanceof Array && !options.include.includes(frameIdentifier)
+}
+
+function getFramesFromID3Body(ID3TagBody, ID3Version, options = {}) {
```

Comment:
I am still a bit confused with frame and tag, what the difference. My understanding is that the frame is the container containing the tag data. Correct?

Reply:
A frame is e.g. `TALB` (album), it contains information and is part of the ID3 tag.
An ID3 tag is the whole buffer containing the ID3 header and its frames.
Sometimes, a set of frames is called tags (not by the specs, but informally), which might be a bit confusing.

There are also parts in the code that are wrongly named from earlier times e.g. https://github.com/Zazama/node-id3/blob/master/src/ID3Util.js#L86 which is named `getFramePosition` even though it searches for the tag's position.

Reply:
OK thanks for the explanation. We should fix same names in a follow up PR.